plugins/Ecommerce/Columns/ProductViewPrice.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+namespace Piwik\Plugins\Ecommerce\Columns;
+
+use Piwik\Plugin\Dimension\ActionDimension;
+use Piwik\Tracker\Action;
+use Piwik\Tracker\Request;
+use Piwik\Tracker\Visitor;
+
+class ProductViewPrice extends ActionDimension
+{
+    protected $type = self::TYPE_MONEY;
+    protected $nameSingular = 'Ecommerce_ViewedProductPrice';
+    protected $columnName = 'product_price';
+    protected $segmentName = 'productViewPrice';
+    protected $columnType = 'DOUBLE NULL';
```

Comment:
not sure if that should be maybe unsigned? I see in other tables it's not unsigned but maybe here it would make sense? not sure... because it's double  and stores big values it might not be needed?

Reply:
Would leave it as signed as in theory someone might maybe want to track a voucher that has a negativ price maybe? 🤔